pkg/install/v1alpha1/jiva_volume.go

```diff
@@ -277,30 +289,25 @@ metadata:
   name: jiva-volume-isvalidversion-default
 spec:
   meta: |
-    id: is070jivavolume
-    runNamespace: {{.Volume.runNamespace}}
+    id: is082jivavolume
+    runNamespace: {{ .Volume.runNamespace }}
     apiVersion: v1
     kind: Service
     action: list
     options: |-
-      labelSelector: vsm={{ .Volume.owner }}
+      labelSelector: openebs.io/controller-service=jiva-controller-svc,openebs.io/version=0.8.2
```

Comment:
we will get all the volumes here.. but, we are supposed to get only for one right?

Reply:
we don't need version check

Reply:
We will be getting list of services but, i think there is some logic written already to get particular svc. @AmitKumarDas can put some comment about how this works internally and regarding the version check, it is done while calling the respective castemplate.